anime_downloader/sites/animeonline360.py

```diff
@@ -0,0 +1,54 @@
+import json, requests
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+class AnimeOnline(Anime, sitename = 'animeonline360'):
+
+    sitename = 'animeonline360'
+
+    @classmethod
+    def search(cls, query):
+        try:
+            # r = helpers.get('https://animeonline360.me/wp-json/dooplay/search/?nonce=12d75e884b', params = {'keyword': query}).json()
+            r = helpers.soupify(helpers.get('https://animeonline360.me/', params = {'s': query})).select('div.title')
+            results = [{"title": x.text, "url": x.a['href']} for x in r]

```

Comment:
This seems redundant, move down one step to SearchResult

Reply:
can you be more specific? do you mean the episode titles or the commented line of code? if its the latter then i will remove it.

Reply:
Instead of saving it to a variable and then using it you can use it directly in searchreuslts like this. 
```
            search_results = [
                SearchResult(
                    title = i.text,
                    url = i.a['href'],
                    meta= {})
                for i in r
            ]
```

Reply:
I see no problem with the way i have written it. So i wont change it.

Reply:
So that i can resolve this comment i will say this:
The code works as intended, it is not bad and it is 100% readable. There is no reason to change it just because you prefer another way to achieve the same exact result. There is no profit either by changing the code.